speech/cloud-client/transcribe_streaming.py

```diff
@@ -29,20 +29,30 @@
 def transcribe_streaming(stream_file):
     """Streams transcription of the given audio file."""
     from google.cloud import speech
-    speech_client = speech.Client()
+    from google.cloud.speech import types
+    client = speech.SpeechClient()
 
     with io.open(stream_file, 'rb') as audio_file:
-        audio_sample = speech_client.sample(
-            stream=audio_file,
-            encoding=speech.encoding.Encoding.LINEAR16,
-            sample_rate_hertz=16000)
-        alternatives = audio_sample.streaming_recognize('en-US')
-
-        for alternative in alternatives:
-            print('Finished: {}'.format(alternative.is_final))
-            print('Stability: {}'.format(alternative.stability))
-            print('Confidence: {}'.format(alternative.confidence))
-            print('Transcript: {}'.format(alternative.transcript))
+        content = audio_file.read()
+        request = types.StreamingRecognizeRequest(audio_content=content)
+
+    config = types.RecognitionConfig(
+        encoding='LINEAR16',
+        sample_rate_hertz=16000,
+        language_code='en-US')
+
+    # In practice requests should be a generator yielding chunks of audio data.
+    requests = [request]
```

Comment:
Might be clearer if you only construct the `StreamingRecognizeRequest` within a generator. ie:

```python
requests = (types.StreamingRecognizeRequest(audio_content=c) for c in [content])
```